app/config/Config.scala

```diff
@@ -37,6 +37,8 @@ class Config(playConfig: Configuration) extends AwsInstanceTags with Logging {
   lazy val mediaAtomMakerUrl: String = s"https://video.$domain"
   lazy val atomWorkshopUrl: String = s"https://atomworkshop.$domain"
 
+  private lazy val recipesUrl: String = s"https://recipes.${if(isProd) Prod.appDomain else Dev.appDomain}"
```

Comment:
Is this different from:
```suggestion
  private lazy val recipesUrl: String = s"https://recipes.$domain"
```